tests/table.spec.ts

```diff
@@ -22,6 +22,10 @@ async function openSimpleTable(page: Page) {
   );
 
   // Model is loaded, need to make sure table data is also loaded
+  await waitForLoadingDone(page);
+}
+
+async function waitForLoadingDone(page: Page) {
```

Comment:
It doesn't really matter, but I'd usually put this function about `openSimpleTable` since `openSimpleTable` calls this function.